cypress/e2e/ModelizerPage/TextView/file/add.file.feature

```diff
@@ -0,0 +1,65 @@
+Feature: Test modelizer text view: add file
+
+  Background:
+    Given I clear cache
+    And I set context field "repository_url" with "https://github.com/ditrit/leto-modelizer-project-test"
+    And I set context field "projectName" with "leto-modelizer-project-test"
+    And I visit the "/"
+
+    When I click on "[data-cy=\"import-project\"]"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-repository-input\"]" text "{{ repository_url }}"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-username-input\"]" text "test"
+    And  I set on "[data-cy=\"import-project-form\"] [data-cy=\"git-token-input\"]" text "test"
+    And  I click on "[data-cy=\"import-project-form\"] [data-cy=\"import-project-form-submit\"]"
+    Then I expect "positive" toast to appear with text "Project has been imported 🥳!"
+    And  I expect "[data-cy=\"import-project-form\"]" is closed
+    And  I expect current url is "modelizer/{{ projectName }}/model"
+    And  I expect "[data-cy=\"project-name\"]" is "{{ projectName }}"
+    And  I visit the "/#/modelizer/{{projectName}}/text"
+    And  I expect "[data-cy=\"file-explorer\"] [data-cy=\"file-label-{{ projectName }}\"]" exists
+    When I wait 2 seconds
+
+  Scenario: An unmodified file should not have the "add" action inside file explorer menu
+    When I click on "[data-cy=\"file-explorer\"] [data-cy=\"file-label-{{ projectName }}\"]"
+    And  I hover "[data-cy=\"file-explorer\"] [data-cy=\"file-explorer-buttons-README.md\"]" to make it visible and click on it
+    Then I expect "[data-cy=\"file-explorer-action-menu\"]" exists
+    And  I expect "[data-cy=\"file-explorer-menu-add-file\"]" not exists
+    And  I expect "[data-cy=\"file-label-README.md\"]" to have "file-status-unmodified" class
+
+  Scenario: Create a file inside the root folder and add it on git should change the file's class
+    When I hover "[data-cy=\"file-explorer\"] [data-cy=\"file-explorer-buttons-{{ projectName }}\"]" to make it visible and click on it
```

Comment:
Before startingthe scenario, verify that the file status

Reply:
I don't understand, I create a file so I cannot verify its status